src/node/db/AuthorManager.js

```diff
@@ -181,41 +185,42 @@ exports.createAuthor = async (name) => {
 
 /**
  * Returns the Author Obj of the author
- * @param {String} author The id of the author
+ * @param {String} authorID The id of the author
```

Comment:
Nit: Use `authorId` instead of `authorID`. Rationale: Style guides used by many companies require lowercase `d` because it facilitates conversion to snake case by tools that bridge languages (every uppercase letter is converted to an underscore followed by that letter in lowercase). We don't use such tools (yet), but consistency with those style guides has value.

Reply:
you mean only in the documentation? won't that cause jsdoc etc to complain that it does not know the signature?
but anyway, can do.

Reply:
Both documentation and code.